src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import url(_variables.scss);
+
+$field--background-color: #eee;
+$field--hover-color: pink;
+
 body {
   margin: 0;
 }
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 9px;
```

Comment:
Improve it 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/e3f9381d-a44b-4c9a-98f9-2d0aad08a5ef)
```suggestion
```

Use margins for mix class or for body tag in your situation